HTML/mockup-mogo/index.html

```diff
@@ -0,0 +1,1024 @@
+                    <p class="offer-item__paragraph paragraph">
+                        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.
+                    </p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <section class="section-design">
+        <div class="section-design__content">
+            <h4 class="subheading subheading--section">For all devices</h4>
+            <h3 class="heading heading--section">Unique design</h3>
+        </div>
+        <picture class="section-design__image-tab">
+            <source media="(min-width: 800px)" srcset="img/display-bg-768-x-1024@3x-min.png">
+            <source media="(min-width: 500px)" srcset="img/display-bg-768-x-1024@2x-min.png">
+            <img src="img/display-bg-768-x-1024-min.png" alt="tablet-photo">
+        </picture>
+        <picture class="section-design__image-phone">
+            <source media="(min-width: 800px)" srcset="img/white@3x-min.png">
+            <source media="(min-width: 500px)" srcset="img/white@2x-min.png">
+            <img src="img/white-min.png" alt="phone-photo">
+        </picture>
+    </section>
+
+    <section class="container section-range">
+        <div class="section-range__content">
+            <h4 class="subheading subheading--section">Service</h4>
+            <h3 class="heading heading--section">what we do</h3>
+            <p class="section-range__paragraph paragraph">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore
+                et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+                aliquip ex ea commodo consequat.
+            </p>
+        </div>
+        <div class="section-range__container">
+            <picture class="section-range__image">
+                <source media="(min-width: 800px)" srcset="img/layer-43@3x-min.png">
+                <source media="(min-width: 500px)" srcset="img/layer-43@2x-min.png">
+                <img src="img/layer-43-min.png" alt="man-photo">
+            </picture>
+            <div class="section-range__options">
+                <div class="option-item">
+                    <div class="option-item__top">
+                        <div class="option-item__content">
+                            <picture class="option-item__icon">
+                                <source media="(min-width: 800px)" srcset="img/picture@3x.png">
+                                <source media="(min-width: 500px)" srcset="img/picture@2x.png">
+                                <img src="img/picture.png" alt="picture-icon">
+                            </picture>
+                            <h4 class="label">Photography</h4>
+                        </div>
+                        <button type="button" name="arrow" class="option-item__btn btn btn--arrow">
+                            <picture class="option-item__arrow">
```

Comment:
not sure if it is okay, to have picture in button, also there is a tag `<input type="image">` for things like this